README.md

```diff
@@ -0,0 +1,13 @@
+# Fyne.io Website
+
+This repository contains the source of the main https://fyne.io website.
+
+## Running the website locally
+
+The website uses [jekyll](https://jekyllrb.com/) to generate the website from markdown files.
+With ruby installed it is as easy as running the following commands in a terminal:
+```bash
+gem install bundler jekyll
+jekyll new my-awesome-site
```

Comment:
I’m pretty sure this line should not be here, as the site is already initialised

Reply:
Yeah sorry, looks like forgot to remove that line. Will get it fixed tomorrow.